spec/signup_spec.rb

```diff
@@ -132,16 +132,18 @@
 		expect(page.status_code).to eq(400)
 	end
 
-  it "should require a @location.police.uk or @police.uk email for person_email" do
-		visit '/signup'
-		fill_in('person_name', with: 'jeff')
-		fill_in('person_email', with: 'jeff@notthepolice.uk')
-		fill_in('department_name', with: 'TestDept')
-		fill_in('service_name', with: 'TestService')
-		click_button('signup-submit')
-		expect(page.first('.form-group--person_email .error-message').text).not_to be_empty
-		expect(page.status_code).to eq(400)
-	end
+  %w(gov.uk nhs.net nhs.uk mod.uk police.uk).each do |tld|
```

Comment:
I think we could add a second test here for accepting subdomains of our allowed list of domains, since it'd be easy to cause a regression in that when editing the regular expressions

Reply:
done in 9b575bf